src/commands/studio.ts

```diff
@@ -295,94 +295,85 @@ export class StudioActions {
       ? [type.toString(), action.id, this.name, answer, msg]
       : [type.toString(), action.id, this.name, selectedText];
 
-    return vscode.window.withProgress(
-      {
-        cancellable: false,
-        location: vscode.ProgressLocation.Notification,
```

Comment:
Rather than removing the progress indicator did you experiment with changing its location to SourceControl or Window?

Reply:
I did try the SourceControl location. It was much less invasive (just a blue clock badge on the source control view icon). We don't actually populate that view though so I didn't think it made sense to call any attention to it. I didn't try the Window location. I think that most actions are very "in your face" (showing a modal dialog or web page etc) so we don't need another visual indicator.

Reply:
Here's what the Window progress location looks like:

<img width="739" alt="Screenshot 2024-02-20 at 7 46 43 AM" src="https://github.com/intersystems-community/vscode-objectscript/assets/44776135/781a2002-11cf-4a03-a62a-b76a310b534a">

I think this is fine as a compromise.